Source/WebCore/platform/mac/PlatformPasteboardMac.mm

```diff
@@ -375,7 +375,7 @@ static String typeIdentifierForPasteboardType(const String& pasteboardType)
         return 0;
 
     NSArray *urlWithTitle = @[ @[ urlString ], @[ pasteboardURL.title ] ];
-    NSString *pasteboardType = [NSString stringWithUTF8String:WebURLsWithTitlesPboardType];
+    NSString *pasteboardType = [NSString stringWithCString:WebURLsWithTitlesPboardType encoding:NSASCIIStringEncoding];
```

Comment:
Some changes in this PR are just syntax, and others are a little harder to reason about. It would be best to either split it into mechanical and non-mechanical parts, or to have specific explanations in the commit message.

For example, this one probably reduces efficiency (more argument to pass, and I'm not sure if conversion from NSASCIIStringEncoding to UTF-8 is actually free, there are often edge cases to deal with unexpected input).

Reply:
I mean PBOARDTYPE is literally an ASCIILiteral type

Reply:
I understand that the string is ASCII only, however this doesn't necessarily mean that using `NSASCIIStringEncoding` to convert it into NSString is higher performing or more correct in any way. I suspect that it's the opposite.